schemas/com.snowplowanalytics.snowplow.media/volume_change_event/jsonschema/1-0-0

```diff
@@ -0,0 +1,29 @@
+{
+  "$schema": "http://iglucentral.com/schemas/com.snowplowanalytics.self-desc/schema/jsonschema/1-0-0#",
+  "description": "Media player event sent when the volume has changed.",
+  "self": {
+    "vendor": "com.snowplowanalytics.snowplow.media",
+    "name": "volume_change_event",
+    "format": "jsonschema",
+    "version": "1-0-0"
+  },
+  "type": "object",
+  "properties": {
+    "previousVolume": {
+      "type": [
+        "integer",
+        "null"
+      ],
+      "description": "Volume percentage before the change.",
+      "minimum": 0,
+      "maximum": 100
+    },
+    "newVolume": {
+      "type": "integer",
```

Comment:
Same for `newVolume` -- I noticed you don't allow null, so I wonder if you also intended to make it required?